lib/src/components/NxTabs/NxTabs.scss

```diff
@@ -16,13 +16,17 @@
 
   border-bottom: var(--nx-border-default);
   display: flex;
-  margin: 0;
+  margin: 0 0 var(--nx-spacing-6x) 0;
   padding: 0;
 }
 
 .nx-tab-panel {
   @include nx-container-helpers.container-vertical;
-  padding-top: var(--nx-spacing-6x);
+
+  &:focus {
```

Comment:
I'd say we need designs for the intended focus styling here

Reply:
> I'd say we need designs for the intended focus styling here

Yup! I believe Jason had updated the design in Kaleido